app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    load_dotenv()
+    api_key = os.getenv("API_KEY")
+    url = f"http://api.weatherapi.com/v1/current.json?key={api_key}"
+    filtering = "Paris"
+
+    print(f"Performing request to Weather API for city {filtering}...")
+    response = requests.get(url + f"&q={filtering}")
```

Comment:
Consider adding error handling for the `requests.get` call. This will help manage potential issues such as network errors or invalid responses from the API.